server/test/org/apache/cloudstack/network/lb/CertServiceTest.java

```diff
@@ -348,90 +348,91 @@ public void runUploadSslCertNoRootCert() throws IOException, IllegalAccessExcept
     @Test
     public void runUploadSslCertBadPassword() throws IOException, IllegalAccessException, NoSuchFieldException {
 
-        String certFile = URLDecoder.decode(getClass().getResource("/certs/rsa_self_signed_with_pwd.crt").getFile(),Charset.defaultCharset().name());
-        String keyFile = URLDecoder.decode(getClass().getResource("/certs/rsa_self_signed_with_pwd.key").getFile(),Charset.defaultCharset().name());
-        String password = "bad_password";
+        final String certFile = URLDecoder.decode(getClass().getResource("/certs/rsa_self_signed_with_pwd.crt").getFile(),Charset.defaultCharset().name());
+        final String keyFile = URLDecoder.decode(getClass().getResource("/certs/rsa_self_signed_with_pwd.key").getFile(),Charset.defaultCharset().name());
+        final String password = "bad_password";
```

Comment:
Please see line notes for 154-155 as those comments apply to lines 351-353 as well.